src/Makefile.am

```diff
@@ -51,8 +51,8 @@ pyTESTS_HANGING = \
 pyTESTS = $(pyTESTS_PASSING) $(pyTESTS_FAILING)
 
 .PHONY: pycheck
-pycheck: cockpit-bridge.pyz
-	$(MAKE) check XFAIL_TESTS='$(pyTESTS_FAILING)' TESTS='$(pyTESTS)' COCKPIT_BRIDGE=./cockpit-bridge.pyz
+pycheck: $(SYSTEMD_CTYPES_STAMP)
+	$(MAKE) check XFAIL_TESTS='$(pyTESTS_FAILING)' TESTS='$(pyTESTS)' COCKPIT_BRIDGE='python3 -m cockpit.bridge'
```

Comment:
With that, we have no tests any more for the built .pyz. As long as we build that, there should IMHO still be at least one simple test that runs it. (I have some faint memory that you wanted to stop building it at some point, but don't we still need it to send over the wire in the injector?)

Reply:
I.e. how about doing what you do, and additionally running `dist/base1/test-echo.html` with the .pyz?

Reply:
I actually want to drop the `.pyz`.

Reply:
OK, but this PR doesn't, and HACKING even still advertises testing it. We also still have the pybridge scenario in test/run. Hence, I'd either like this PR to add a test-echo smoketest for the .pyz,  or drop it all in one go.